packages/client-core/src/common/services/NotificationService.tsx

```diff
@@ -58,9 +59,18 @@ export const defaultAction = (key: SnackbarKey, content?: React.ReactNode) => {
     </Fragment>
   )
 }
+export const inviteActions = (key: SnackbarKey, content?: React.ReactNode) => {
```

Comment:
instead of creating new action can we move this custom component in the notification itself? `NotificationService.message` accepts `React` component. Line 39 of `useDomainProvisionListener` can be used for reference

Reply:
If I don't add a new action, it will take the current implementation and have a button to close the notification. I can't even add functionality to this action to decline the invitation, it only closes the notification, which doesn't fulfill the expected functionality.

https://github.com/ir-engine/ir-engine/blob/267035aae831b5dec33d705ec44fb432d4684493/packages/client-core/src/common/services/NotificationService.tsx#L56